burrito-fg/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+use breadx::{
+    auto::xproto::ATOM_WM_TRANSIENT_FOR,
+    traits::{PropMode, PropertyFormat, PropertyType, ATOM_WM_NAME},
+    Display, DisplayBase, DisplayConnection, Window,
+};
+use gdnative::prelude::*;
+
+#[derive(NativeClass)]
+#[inherit(Node)]
+pub struct X11_FG;
+// Function that registers all exposed classes to Godot
+fn init(handle: InitHandle) {
+    handle.add_class::<X11_FG>();
+}
+
+impl X11_FG {
+    /// The "constructor" of the class.
+    fn new(_owner: &Node) -> Self {
+        X11_FG
+    }
+}
+
+#[methods]
+impl X11_FG {
+    #[export]
+    fn _ready(&self, _owner: &Node) {
+        godot_print!("Hello,X11!");
+        return;
+    }
+
+    #[export]
+    fn set_transient_for(&self, _owner: &Node, burrito_id: i32) {
```

Comment:
Can we add a comment here as to why we are taking in an `i32` only to immediately cast it to a `u32`. I'm guessing it has something to do with the gdnative api but it would be nice to have a clear record here.

Reply:
Its because the gdnative api allows only types from godot to be given to a method.

Reply:
Makes sense to me.